Formula/sipp.rb

```diff
@@ -22,4 +21,8 @@ def install
     system "make", "DESTDIR=#{prefix}"
     bin.install "sipp"
   end
+
+  test do
+    assert_equal "SIPp", shell_output("#{bin}/sipp -v | grep SIPp | awk '{ print $1 }'").strip
```

Comment:
Confirmed: `assert_match "SIPp v#{version}", shell_output("#{bin}/sipp -v 2>&1 || true")` passes the test.